ui/ceval/src/view.ts

```diff
@@ -27,6 +27,7 @@ function localEvalInfo(ctrl: ParentCtrl, evs: NodeEvals): Array<VNode | string>
     const localEvalText = ceval.isLoaded()
       ? trans.noarg('calculatingMoves')
       : trans.noarg('loadingEngine') + (mb >= 1 ? ` (${mb.toFixed(1)} MiB)` : '');
+    ('');
```

Comment:
This is probably unintentional?

Reply:
that right there is the secret ingredient my friend.